resources/js/Components/navigation/Footer.vue

```diff
@@ -0,0 +1,95 @@
+                    </ul>
+                    <ul class="hidden sm:block">
+                        <MenuItem isFooterMenu="true" name="Certificate" />
+                        <ul class="text-m-p-s space-y-4 mt-4 text-neutral font-medium md:text-d-p-s">
+                            <MenuItem class="hover:underline" none="true" name="Environmental Management" link="/environmental-management" />
+                            <MenuItem class="hover:underline" none="true" name="Occupation Health and Safety Management"
+                                link="/occupation-safety-management" />
+                            <MenuItem class="hover:underline" none="true" name="Personal Professional Development"
+                                link="/professional-development" />
+                            <MenuItem class="hover:underline" none="true" name="Quality Management" link="/quailty-management" />
+                            <MenuItem class="hover:underline" none="true" name="Information Security Management"
+                                link="/info-security-m anagement" />
+                            <MenuItem class="hover:underline" none="true" name="Food Safety Management" link="/food-safety-management" />
+                        </ul>
+                    </ul>
+                    <div class="sm:hidden">
+                        <SubMenuItem uniqueID=certificate-footer name="Certification" isFooterSubmenu="true">
+                            <MenuItem isDropdown="true" name="Environmental Management" link="/environmental-management" />
+                            <MenuItem isDropdown="true" name="Occupation Health and Safety Management" link="/occupation-safety-management" />
+                            <MenuItem isDropdown="true" name="Personal Professional Development" link="/professional-development" />
+                            <MenuItem isDropdown="true" name="Quality Management" link="/quailty-management" />
+                            <MenuItem isDropdown="true" name="Information Security Management"  link="/info-security-m anagement" />
+                            <MenuItem isDropdown="true" name="Food Safety Management" link="/food-safety-management" />
+                        </SubMenuItem>
+                    </div>
+                    <ul class="hidden sm:block">
+                        <MenuItem isFooterMenu="true" name="Services" />
+                        <ul class="text-m-p-s space-y-4 mt-4 text-neutral font-medium md:text-d-p-s">
+                            <MenuItem class="hover:underline" none="true" link="/news-and-recruitment" name="News and Recruitment" />
+                            <MenuItem class="hover:underline" none="true" link="/document-verification" name="Document Verification" />
+                        </ul>
+                    </ul>
+                    <div class="sm:hidden">
+                        <SubMenuItem uniqueID=services-footer  name="Services" isFooterSubmenu="true">
```

Comment:
when you pass variable please use like html attribute and value
```suggestion
                        <SubMenuItem uniqueID="services-footer"  name="Services" isFooterSubmenu="true">
```